to.etc.domui/src/main/java/to/etc/domui/util/exporters/ResultSetExcelExporter.java

```diff
@@ -70,7 +70,10 @@ public void writeExcel(Workbook workbook, ResultSet rs, String sheetName) {
 					}
 				}
 			}
-			sheet.createFreezePane(0, 1, 0, 2);
+			Cell cell = sheet.createRow(0).createCell(0);
```

Comment:
This should become optional; having this means we cannot just re-import the set?